cmd/cluster-node-tuning-operator/main.go

```diff
@@ -321,6 +334,45 @@ func migratePinnedSingleNodeInfraStatus(cfg *rest.Config, scheme *apiruntime.Sch
 	return nil
 }
 
+func setupFeatureGates(ctx context.Context, config *rest.Config, operatorNamespace string) (featuregates.FeatureGate, error) {
+	// TODO is this version correct?
+	desiredVersion := version.Version
```

Comment:
this is the last thing we need to check. Other than this, LGTM - dunno of better ways to achieve the same result (in the required time frame)